pkg/volume/local/local.go

```diff
@@ -50,6 +52,11 @@ const (
 
 func (plugin *localVolumePlugin) Init(host volume.VolumeHost) error {
 	plugin.host = host
+	plugin.volumeLocks = keymutex.NewKeyMutex()
+	mntTracker = localVolumeMounts{
```

Comment:
I prefer the method of getting the real mount state from the host, instead of relying on in-memory refcounting because:
1. Like you mentioned, we would need to initialize this with the actual host state in case kubelet restarted.
2. People may log into the node and force mount/unmount outside of the plugin.